tools/accuracy_checker/accuracy_checker/main.py

```diff
@@ -38,145 +38,266 @@
 
 def build_arguments_parser():
     parser = ArgumentParser(description='Deep Learning accuracy validation framework', allow_abbrev=False)
-    parser.add_argument(
+    common_args = parser.add_argument_group('Common arguments')
+    common_args.add_argument(
         '-d', '--definitions',
         help='path to the yml file with definitions',
         type=get_path,
         required=False
     )
-    parser.add_argument(
+    common_args.add_argument(
         '-c', '--config',
         help='path to the yml file with local configuration',
         type=get_path,
         required=True
     )
-    parser.add_argument(
+    common_args.add_argument(
         '-m', '--models',
         help='prefix path to the models and weights',
         type=partial(get_path, is_directory=True),
         required=False,
         nargs='+'
     )
-    parser.add_argument(
+    common_args.add_argument(
         '-s', '--source',
         help='prefix path to the data source',
         type=partial(get_path, is_directory=True),
         required=False
     )
-    parser.add_argument(
+    common_args.add_argument(
         '-a', '--annotations',
         help='prefix path to the converted annotations and datasets meta data',
         type=partial(get_path, is_directory=True),
         required=False
     )
-    parser.add_argument(
-        '-e', '--extensions',
-        help='prefix path to extensions folder',
-        type=partial(get_path, check_exists=False),
+    common_args.add_argument(
+        '-C', '--converted_models',
+        help='directory to store Model Optimizer converted models. Used for DLSDK launcher only',
+        type=partial(get_path, is_directory=True),
         default=Path.cwd(),
         required=False
     )
-    parser.add_argument(
-        '--cpu_extensions_mode',
-        help='specified preferable set of processor instruction for automatic searching cpu extension lib',
+    common_args.add_argument(
+        '-tf', '--target_framework',
+        help='framework for infer',
+        required=False
+    )
+    common_args.add_argument(
+        '-td', '--target_devices',
+        help='space separated list of devices for infer',
         required=False,
-        choices=['avx512', 'avx2', 'sse4']
+        nargs='+'
     )
-    parser.add_argument(
-        '-b', '--bitstreams',
-        help='prefix path to bitstreams folder',
-        type=partial(get_path, file_or_directory=True),
+    common_args.add_argument(
+        '-tt', '--target_tags',
+        help='space separated list of launcher tags for infer',
+        required=False,
+        nargs='+'
+    )
```

Comment:
I think it should be added to separated group - filtration

Reply:
Added